.github/workflows/test.yml

```
@@ -0,0 +1,24 @@
+name: CI
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+on:
+  pull_request:
+  push:
+    branches: [main]
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test-npm-package:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 22
```

```suggestion
        node-version: 18
```

We should test against Netlify's current default Node version, which is 18 (though you could convert this to a matrix test if you want to test against multiple versions!)